pandas/tests/computation/test_eval.py

```diff
@@ -1625,7 +1641,26 @@ def eval(self, *args, **kwargs):
     def test_unary_functions(self):
         df = DataFrame({'a': np.random.randn(10)})
         a = df.a
-        for fn in self.unary_fns:
+        unary_functions = [x for x in self.unary_fns
```

Comment:
can you parameterize this over the unary functions (it will automatically just work according to the version of numexpr)